README.md

```diff
@@ -85,7 +132,10 @@ const results = await spectral.run({
   name: 'helloWorld',
```

Comment:
To be removed :) We have a new code below.

Reply:
Lol dang it, you beat me before I could push that fix. :P